src/NC-State-Parks.ino

```diff
@@ -259,13 +262,11 @@ void setup()                                        // Note: Disconnected Setup(
     resetEverything();                                                 // Zero the counts for the new day
   }
 
-  if ((Time.hour() >= sysStatus.closeTime || Time.hour() < sysStatus.openTime)) {   // The park is closed - don't connect unless we need to set the RTC
-    if (!ab1805.isRTCSet()) connectToParticle();                      // We need to connect and set the clock
-    Particle.syncTime();
-  } 
-  else {                                                              // Park is open let's get ready for the day
+  if ((Time.hour() >= sysStatus.openTime) && (Time.hour() < sysStatus.closeTime)) { // Park is open let's get ready for the day                                                            
     attachInterrupt(intPin, sensorISR, RISING);                       // Pressure Sensor interrupt from low to high
-    if (sysStatus.connectedStatus && !Particle.connected()) connectToParticle(); // Only going to connect if we are in connectionMode
+    if (sysStatus.connectedStatus && !Particle.connected()) {         // If the system thinks we are connected, let's make sure that we are
```

Comment:
This line still feels odd to me - @chipmc, you mentioned it was there in case of an edge case regarding resets/shutdowns?

Reply:
Yes, the worse event for one of these devices is that it goes off-line.  This line ensures that if the device is reset or switched on during waking hours that it will connect to Particle.  We could take it out but then we would need to make sure we could bring the device on-line - perhaps by pressing the user button.

Reply:
I am OK with taking this line out for now and then playing with it locally to see the implications.

Reply:
I added a comment on the next line to try to give it some more context, but if you have found it useful in the past, let's leave it here for now.